app/uk/gov/hmrc/gform/views/hardcoded/pages/confirm_delete.scala.html

```diff
@@ -7,7 +7,7 @@
 
 @uk.gov.hmrc.gform.views.html.main_template(title = "Confirm Delete", bodyClasses = None) {
 
-<form action="@uk.gov.hmrc.gform.controllers.routes.FormController.delete(formTypeId, formId)" method="POST">
+<form action="@uk.gov.hmrc.gform.controllers.routes.FormController.delete(formTypeId, formId)" method="GET">
```

Comment:
https://softwareengineering.stackexchange.com/questions/114156/why-are-there-are-no-put-and-delete-methods-on-html-forms DELETE can not be supported in the form in HTML5